posthog/models/insight.py

```diff
@@ -44,6 +46,7 @@ class Insight(RootTeamMixin, FileSystemSyncMixin, models.Model):
     filters = models.JSONField(default=dict)
     filters_hash = models.CharField(max_length=400, null=True, blank=True)
     query = models.JSONField(null=True, blank=True)
+    query_metadata = models.JSONField(default=dict, null=True, blank=True)
```

Comment:
I'd prefer if this migration was split out and merged first, sometimes our migration timing makes things tricky and it lowers any chances of issues.